### DISCUSSION TITLE
is there any way to disable just the '✅ no new version available' message?

### DISCUSSION THREAD

**Discussion by jbgreenh** [JanDeDobbeleer/oh-my-posh#5345] [Troubleshoot] [answered]
hello i recently added `auto_upgrade = true` to my config and i get the '✅ no new version available' message most times i open up my shell. what i am wondering is if there is a way to disable this message but still show when upgrades are being done?

as a side note, i tried running `oh-my-posh disable notice` thinking it would disable both completely (while i would prefer the way i described above, i would prefer no notice at all over the '✅ no new version available' message) and it doesn't seem to hide this message:
![image](https://github.com/user-attachments/assets/43262ab8-3fd9-4b91-8f75-994ee15e5a62)

any advice for how i can accomplish this? any and all help would be appreciated.

**Comment by JanDeDobbeleer** [comment, marked as answer]
`disable_notice: true`

**Reply by jbgreenh** [reply]
awesome thank you!
